terraform/devnet-0/main.tf

```diff
@@ -440,10 +440,15 @@ resource "local_file" "ansible_inventory" {
             hostname = "${split(".", key)[0]}-${split(".", key)[1]}"
             cloud    = "digitalocean"
             region   = "${server.region}"
+            ansible_vars = try(
+              var.digitalocean_vm_groups[
+                index([for v in var.digitalocean_vm_groups : v.id], split(".", key)[0])
+              ].vms[split(".", key)[1]].ansible_vars
+            , null)
           }
         }
       )
     }
   )
-  filename = "../../ansible/inventories/devnet-0/inventory.ini"
+  filename = "../../ansible/inventories/devnet-2m-test/inventory.ini"
```

Comment:
We shouldn’t change the defaults

Reply:
agreed, made a mistake backporting fixes. fixed